src/EditorFeatures/Core/EditAndContinue/ActiveStatementTaggerProvider.cs

```diff
@@ -40,10 +40,6 @@ internal partial class ActiveStatementTaggerProvider(
     [Import(AllowDefault = true)] ITextBufferVisibilityTracker? visibilityTracker,
     IAsynchronousOperationListenerProvider listenerProvider) : AsynchronousTaggerProvider<ITextMarkerTag>(threadingContext, globalOptions, visibilityTracker, listenerProvider.GetListener(FeatureAttribute.Classification))
 {
-    // We want to track text changes so that we can try to only reclassify a method body if
-    // all edits were contained within one.
-    protected override TaggerTextChangeBehavior TextChangeBehavior => TaggerTextChangeBehavior.TrackTextChanges;
```

Comment:
this was never used.  this asked the tagger infrastructure to track this info.  but then it was never used in this tagger.

Reply:
Note: it's clear that this was just a copy/paste from the semantic-classifier